src/core/InstrumentFunctions.cpp

```diff
@@ -569,13 +569,10 @@ void InstrumentFunctionArpeggio::loadSettings( const QDomElement & _this )
 	// Keep compatibility with version 0.2.1 file format
 	if( _this.hasAttribute( "arpsyncmode" ) )
 	{
-	 	m_arpTimeKnob->setSyncMode( 
+	 	m_arpTimeKnob->setSyncMode(
  		( tempoSyncKnob::tempoSyncMode ) _this.attribute(
  						 "arpsyncmode" ).toInt() );
 	}*/
 
 	m_arpModeModel.loadSettings( _this, "arpmode" );
 }
-
-
-
```

Comment:
No. 4 spaces fine.

Reply:
I'll restore them

Reply:
What does No.4 spaces mean?

Reply:
It means I was somewhere else in the code... Sorry!